hack/deploy-hub-local/sno-metal3.yml

```diff
@@ -0,0 +1,23 @@
+version: stable
+tag: 4.10
+sno: true
```

Comment:
as an alternative, you can set masters to 1 and dont set sno to true, which will deploy using a bootstrap vm.
This workflow allows you to install extra operators through the apps  array